unity-client/Assets/Scripts/MainScripts/DCL/Components/Video/Tests/VideoTests.cs

```diff
@@ -163,15 +165,161 @@ public IEnumerator VideoTextureVisibleStateIsSetCorrectlyWhenEntityIsRemoved()
             yield return ent1Shape.routine;
 
             TestHelpers.SharedComponentAttach(ent1Shape, ent1);
-            yield return null; //a frame to wait DCLVideoTexture update    
+            yield return null; //a frame to wait DCLVideoTexture update
             Assert.IsTrue(videoTexture.isVisible, "DCLVideoTexture should be visible");
 
             scene.RemoveEntity(ent1.entityId, true);
-            yield return null; //a frame to wait DCLVideoTexture update            
+            yield return null; //a frame to wait DCLVideoTexture update
 
             Assert.IsTrue(!videoTexture.isVisible, "DCLVideoTexture should not be visible ");
         }
 
+        //--------------------------------
+
+        [UnityTest]
+        public IEnumerator VolumeWhenVideoCreatedWithNoUserInScene()
+        {
+            DCLCharacterController.i.Teleport(JsonConvert.SerializeObject(new
+            {
+                x = -1f,
+                y = DCLCharacterController.i.transform.position.y,
+                z = -1f
```

Comment:
Cannot we just avoid all the teleporting and rely just in changing the sceneId/playerCoords scriptableObjects?

Reply:
done